lib/template_data/intro.md

```diff
@@ -1,11 +1,116 @@
 # `PySAL` Notebooks Book
 
-This is a compilation of official notebooks demonstrating the functionality of
-`PySAL`, the Python Spatial Analysis library.
+This is a compilation of official notebooks demonstrating the functionality of `PySAL`, the
+Python Spatial Analysis library.
 
----
+* * *
 
-**IMPORTANT**: fill in here with more general information about PySAL and the 
-notebooks project
+# Introduction
 
----
+The PySAL project began began life a decade ago as a collaboration between Serge Rey and
```

Comment:
I'd start with a punch line of what PySAL is and seeks to accomplish. e.g. "PySAL is a comprehensive library implementing spatial analytics in Python.